src/api/db/structure.sql

```diff
@@ -878,26 +878,26 @@ CREATE TABLE `repositories` (
   `db_project_id` int(11) NOT NULL,
   `name` varchar(255) COLLATE utf8_bin NOT NULL,
   `remote_project_name` varchar(255) COLLATE utf8_bin NOT NULL DEFAULT '',
-  `rebuild` enum('transitive','direct','local') CHARACTER SET utf8 DEFAULT NULL,
-  `block` enum('all','local','never') CHARACTER SET utf8 DEFAULT NULL,
-  `linkedbuild` enum('off','localdep','all') CHARACTER SET utf8 DEFAULT NULL,
+  `rebuild` varchar(10) CHARACTER SET utf8 DEFAULT NULL,
+  `block` varchar(5) CHARACTER SET utf8 DEFAULT NULL,
+  `linkedbuild` varchar(8) CHARACTER SET utf8 DEFAULT NULL,
   `hostsystem_id` int(11) DEFAULT NULL,
   PRIMARY KEY (`id`),
-  UNIQUE KEY `projects_name_index` (`db_project_id`,`name`,`remote_project_name`),
-  KEY `remote_project_name_index` (`remote_project_name`),
-  KEY `hostsystem_id` (`hostsystem_id`),
+  UNIQUE KEY `projects_name_index` (`db_project_id`,`name`,`remote_project_name`) USING BTREE,
+  KEY `hostsystem_id` (`hostsystem_id`) USING BTREE,
+  KEY `remote_project_name_index` (`remote_project_name`) USING BTREE,
   CONSTRAINT `repositories_ibfk_1` FOREIGN KEY (`db_project_id`) REFERENCES `projects` (`id`),
   CONSTRAINT `repositories_ibfk_2` FOREIGN KEY (`hostsystem_id`) REFERENCES `repositories` (`id`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_bin;
 
 CREATE TABLE `repository_architectures` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
```

Comment:
Actually this is a trivial change - the id column was just moved up from a few lines below.